Automattic/wp-calypso
# ToS tracking: Encrypted file with bearer token of the user with upload permissions

#### Changes proposed in this Pull Request

* This PR adds to the encrypted file the bearer token of the user that has upload permissions to the ToS blog.
* This bearer token is used in https://github.com/Automattic/wp-calypso/pull/57741/ to upload screenshots to the server.

#### Testing instructions



* pull this branch
* decrypt the secrets
* ensure `martechTosUploadCredentials` exists in the file.
